python/test/unit/tools/test_aot.py

```diff
@@ -98,18 +101,37 @@ def kernel(C, A, B, M, N, K,
 
 
 def gen_kernel_library(dir, libname):
-    c_files = glob.glob(os.path.join(dir, "*.c"))
-    subprocess.run(
-        ["gcc"] + c_files + ["-I", include_dir[0], "-c", "-fPIC"],
-        check=True,
-        cwd=dir,
-    )
-    o_files = glob.glob(os.path.join(dir, "*.o"))
+    if is_cuda():
```

Comment:
May be to minimize changes and simplify future potential merges
```suggestion
    if is_xpu():
       gen_kernel_library_xpu(dir, libname)
```

and keep the rest of the function as the original?

Reply:
Thanks for review. Changed in https://github.com/intel/intel-xpu-backend-for-triton/pull/2953/commits/f6aaed5c0e95d8e1441e9afa561e75e6246baf4b